src/rgw/rgw_sal_motr.cc

```diff
@@ -1528,7 +1528,11 @@ int MotrObject::fetch_obj_entry_and_key(const DoutPrefixProvider* dpp, rgw_bucke
   if (rc < 0) {
       ldpp_dout(dpp, 0) <<__func__<< ": ERROR: failed to get object entry. rc=" << rc << dendl;
       return rc;
-    }
+  }
+  if(ent.is_delete_marker()){
```

Comment:
Suggestion: Plz add space between 'if' and '{'.
E.g:
`if (ent.is_delete_marker()) {`

Reply:
yes, lets try to follow the coding standard for rgw now. recent PR 313 had many such corrections, lets avoid any such changes now

Reply:
Addressed in recent patch